includes/Server.hpp

```diff
@@ -25,5 +25,6 @@ class	Server
 	operator		int();
 	size_t			size();
 
-	ServerData		getData();
+	std::vector<ServerData const>	getData() const;
```

Comment:
here

Reply:
add a const reference on the returned vector